snapcraft.yaml

```diff
@@ -598,8 +598,9 @@ parts:
 
   nvidia-container:
     source: https://github.com/NVIDIA/libnvidia-container
+    # XXX: fails to build if using source-commit
     source-depth: 1
-    source-tag: v1.14.6
+    source-tag: v1.16.1
```

Comment:
you might need to remove source-depth: 1

Reply:
Good idea, that did the trick (https://github.com/canonical/lxd-pkg-snap/pull/557)!